src/guide/faqs.md

```diff
@@ -0,0 +1,100 @@
+---
+title: FAQs
+type: guide
+order: 1
+---
+
+<p class="tip">MachineLabs is currently in private beta. <strong>To follow this guide you need to be part of the private beta program</strong>. If you haven't done so yet, login at <a href="https://machinelabs.ai/editor" title="MachineLabs Editor">machinelabs.ai</a> to become part of the next batch. We onboard new users weekly.</p>
```

Comment:
Can we make this an include so that we keep things more DRY

Reply:
I need to figure out how that is done in hexo

Reply:
Okay I've found https://www.npmjs.com/package/hexo-include

It says the include path is relative to the source path. I tried all possible variations now but I can't make it find the file to include right now.

Will revisit.